go_test.go

```diff
@@ -0,0 +1,171 @@
+package semdiffstat
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+)
+
+func TestGo(t *testing.T) {
+	tests := []struct {
+		fileA   []byte
```

Comment:
You can remove a bunch of line noise by having these be strings and converting to []byte in the loop below as part of calling `Go`.